src/index.js

```diff
@@ -37,6 +37,12 @@ var dbCall = function (clientCtx, config, stackTrace) {
             return _.get(inputParams, m.name);
          });
 
+         _(inputParams).keys().forEach(function (param) {
+            if (_.isNil(_.find(config.mapping, function (m) { return _.split(m.name, /[.\[\]]/)[0] === param }))) {
+               throw new Error('Unused key [' + param + '] in input parameters.')
```

Comment:
I think it would be helpful to find a way to warn instead of error. 

I'm a little confused at how this works at first glance.